libs/ml/include/ml/layers/self_attention_encoder.hpp

```diff
@@ -163,7 +163,7 @@ class SelfAttentionEncoder : public SubGraph<T>
     std::string ff_first_layer =
         this->template AddNode<fetch::ml::layers::FullyConnected<TensorType>>(
             name + "_Feedforward_No_1", {input}, static_cast<SizeType>(model_dim_),
-            static_cast<SizeType>(ff_dim_), ActivationType::RELU, RegType::NONE,
```

Comment:
is this generically correct? or specific to BERT?
seems like this should be an input parameter if its specific to BERT

Reply:
you are right